index.html

```diff
@@ -411,7 +162,7 @@ <h2>About me</h2>
           <img alt="division" src="img/line2.png" class="line2" />
         </section>
         <section class="skills d-flex">
-          <ul>
+          <ul class="tech">
             <li>
               <h2 class="lan">Language</h2>
```

Comment:
-  [ ] [REQUIRED] Kindly ensure that the orange icon beside `Language` has a little space as provided in the template. Doing this will make your design look more like the template provided. Kindly apply this to `Frameworks` and `Skills` also. The `h2` tags.
#### Your Design
![1](https://user-images.githubusercontent.com/58692590/155814968-191296fe-c25e-4a95-b1f6-66d37eae17a0.png)

#### Provided Template
![11](https://user-images.githubusercontent.com/58692590/155814982-78bbb122-e793-461e-8af1-d0464a5d7917.png)